### PR TITLE
Update Ender 4 config

### PR DESCRIPTION
**Pull request by xymopen**
### Description

Applied various settings from [official source](https://www.creality.com/?do_action=action.download&DId=63)

### Benefits

These settings ease the stepper vibration issue on Ender-4, which fix homing and CoreXY kinematics failure.

### CODE REVIEW

File: config/examples/Creality/Ender-4/Configuration.h [hunk 1]
```diff
@@ -491,15 +491,10 @@
 
   // If you are using a pre-configured hotend then you can use one of the value sets by uncommenting it
 
-  // Ender-4
-  #define DEFAULT_Kp 16.67
-  #define DEFAULT_Ki 1.12
-  #define DEFAULT_Kd 62.14
-
```
Comment (thisiskeithb) [inline config/examples/Creality/Ender-4/Configuration.h:498]: Why remove the tuned Ender-4 defaults? You're also restoring Ultimaker defaults below which doesn't make sense here.

File: config/examples/Creality/Ender-4/Configuration.h [hunk 2]
```diff
@@ -803,11 +798,11 @@
...
  */
-//#define CLASSIC_JERK
+#define CLASSIC_JERK
 #if ENABLED(CLASSIC_JERK)
-  #define DEFAULT_XJERK 10.0
-  #define DEFAULT_YJERK 10.0
-  #define DEFAULT_ZJERK  2.4
+  #define DEFAULT_XJERK 20.0
+  #define DEFAULT_YJERK 20.0
+  #define DEFAULT_ZJERK  0.4
```
Comment (thisiskeithb) [inline config/examples/Creality/Ender-4/Configuration.h:811]: Junction Deviation is now the standard, so why revert to Classic Jerk? Also, keep in mind that Classic Jerk settings were cut in half back in Marlin 1.1.8.

File: config/examples/Creality/Ender-4/Configuration_adv.h [hunk 3]
```diff
@@ -979,7 +979,7 @@
  *    M909, M910 & LCD - only PRINTRBOARD_REVF & RIGIDBOARD_V2
  */
 //#define PWM_MOTOR_CURRENT { 1300, 1300, 1250 }          // Values in milliamps
-//#define DIGIPOT_MOTOR_CURRENT { 135,135,135,135,135 }   // Values 0-255 (RAMBO 135 = ~0.75A, 185 = ~1A)
+#define DIGIPOT_MOTOR_CURRENT { 135,135,135,135,135 }   // Values 0-255 (RAMBO 135 = ~0.75A, 185 = ~1A)
 //#define DAC_MOTOR_CURRENT_DEFAULT { 70, 80, 90, 80 }    // Default drive percent - X, Y, Z, E axis
```
Comment (thisiskeithb) [inline config/examples/Creality/Ender-4/Configuration_adv.h]: This doesn't apply to Crealtiy Melzi boards.

**Comment by thisiskeithb** [comment]
Original/OEM configs can be questionable (especially older ones from Creality), but I left several comments above to be reviewed.

File: config/examples/Creality/Ender-4/Configuration.h (hunk 2 above)
Comment (thinkyhead) [reply config/examples/Creality/Ender-4/Configuration.h:811]: Classic Jerk may be better in-general for Cartesians. It's still an open question.

**Comment by xymopen** [comment]
I'm sorry I cannot answer these questions. I'm new to Marlin firmware and know little about engineering. These configs do fix some problem from OEM firmware and previous commit Marlin configuration. I thought it's good to make these changes back to the community, so someone know more than me can make a better one.

**Pull request merged**